### PR TITLE
archivebox: init at 0.6.2

### LINKED ISSUE #122793: New Package Request: ArchiveBox
**Project description**

ArchiveBox is a self-hosted internet archiving solution that lets you save sites offline.

**Metadata**

- homepage URL: https://archivebox.io
- source URL: https://github.com/ArchiveBox/ArchiveBox
- license: MIT
- platforms: [Ubuntu/Debian](https://github.com/ArchiveBox/debian-archivebox), [Arch](https://aur.archlinux.org/packages/archivebox/), [FreeBSD](https://github.com/ArchiveBox/ArchiveBox/blob/dev/bin/setup.sh), [macOS](https://github.com/ArchiveBox/homebrew-archivebox), [Windows](https://archivebox.io/#quickstart) (WSL/WSL2/Docker only)

**Dependencies**

- `python` >= 3.7
- `django` >=3.1,<3.2
- `node` >= 12
- `chromium` / `chrome` >= 70
- `wget` >= 1.2
- `curl` >= 7.3
- `youtube-dl`
- `git`

For more info, see: https://github.com/ArchiveBox/ArchiveBox#dependencies and https://github.com/ArchiveBox/ArchiveBox/wiki/Install#1-install-dependencies

**Context**

I'm the maintainer of ArchiveBox, but I'm a NixOS n00b so I don't feel qualified to package it for Nix, but I'd be happy to help anyone that wants to.

### PR DESCRIPTION
**Pull request by vojta001**
###### Motivation for this change
Resolve #122793

Currently broken because of django version

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/backup/archivebox/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchFromGitHub, python3Packages, }:
```
Comment (dotlambda) [inline pkgs/applications/backup/archivebox/default.nix:1]:
```suggestion
{ lib, fetchFromGitHub, python3 }:
```
facilitates overriding

File: pkgs/applications/backup/archivebox/default.nix [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+python3Packages.buildPythonApplication rec {
```
Comment (dotlambda) [inline pkgs/applications/backup/archivebox/default.nix:3]:
```suggestion
python3.pkgs.buildPythonApplication rec {
```

**Review by dotlambda — COMMENTED** [review]
Overriding the version of django being used isn't hard. Look at e.g. home-assistant.

**Comment by mweinelt** [comment]
Superseded by https://github.com/NixOS/nixpkgs/pull/133476.

**Pull request closed without merging**